### ISSUE TITLE
Download Tracks or Albums 

### ISSUE THREAD

**Issue by rmdes** [blackcandy-org/blackcandy#405]
I'm tempting to replace my usage of Funkwhale by your neat and simple approach for web media player but I was wondering if a download track or download album was in the roadmap?

keep up the good work 🙏🏽

**Comment by aidewoode** [comment]
Yes, I have the plan to let users download tracks on mobile app to offer offline listening experience.  But this feature may not come soon.
